src/views/ApartmentApp/components/StudentApplication/index.js

```diff
@@ -278,6 +281,7 @@ const StudentApplication = ({ userProfile, authentication }) => {
       // The method is separated from callback because the housing API service must be handled inside an async method
       changeApplicationEditor(applicationID, newEditorProfile.AD_Username);
       handleCloseOkay();
+      setUnsavedChanges(true);
```

Comment:
Is it necessary to set it here when it is also set in changeApplicationEditor?

Reply:
That is actually a good point, especially because it would cause it to mark it as unsaved changes even if the `changeApplicationEditor` fails

Reply:
This change has now been committed